content/docs/deploy/k8s/ingress.md

```diff
@@ -809,6 +811,37 @@ spec:
       secretName: mcp-tls
 ```
 
+#### MCP Server with Custom Path
+
+If your upstream client application makes use of [MCP Routes Enumeration](https://main.docs.pomerium.com/docs/capabilities/mcp#listing-available-mcp-servers) to determine the exact URL the MCP Streaming HTTP is exposed at, you may specify it with this annotation.
```

Comment:
Change this to a relative link. can backport this whole PR to 0.30, and it will still work, because that section is in the 0.30 release.